src/_includes/partials/navigation.njk

```diff
@@ -0,0 +1,17 @@
+<nav id="app-navigation" class="app-navigation js-app-navigation govuk-clearfix" role="navigation" aria-labelledby="app-navigation-heading">
```

Comment:
Could probably use MOJ's [Primary Navigation component](https://design-patterns.service.justice.gov.uk/components/primary-navigation/) if you wanted

Reply:
Great idea, will move this to another ticket and do that next week, thanks!